src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{

```

Comment:
There should be a space between the class name `Machine` and the curly brace `{` to adhere to Java coding standards.